vispy/gloo/wrappers.py

```diff
@@ -428,29 +432,29 @@ def set_sample_coverage(self, value=1.0, invert=False):
     #
 
     def get_state_presets(self):
-        """The available GL state presets
+        """The available GL state :data:`presets <.GL_PRESETS>`.
 
         Returns
         -------
         presets : dict
-            The dictionary of presets usable with ``set_options``.
+            The dictionary of presets usable with :func:`.set_state`.
         """
-        return deepcopy(_gl_presets)
+        return deepcopy(GL_PRESETS)
 
     def set_state(self, preset=None, **kwargs):
-        """Set OpenGL rendering state, optionally using a preset
+        """Set the OpenGL rendering state, optionally using a preset.
 
         Parameters
         ----------
-        preset : str | None
-            Can be one of ('opaque', 'translucent', 'additive') to use
-            use reasonable defaults for these typical use cases.
+        preset : {None, 'opaque', 'translucent', 'additive'}
```

Comment:
Sorry, I meant in the docstring for the `GL_PRESETS` constant. I think technically everything after the `: ` in a numpy-style docstring like this has to be a type (`str | None`).

Reply:
Ah, OK. I think I got it now. I'll do that.

And you suggest resetting to `preset : str | None` in `set_state()` then?

Reply:
For the docstrings, I don't know. I was following [this source](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), which says

```
When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:

order : {'C', 'F', 'A'}
   Description of `order`.
```

Reply:
You're right. I wasn't sure, but that example is exactly like our case so :+1: